socialhome/streams/app/components/StreamElement.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div>
+        <p v-html="post.content"></p>
+        <div class="grid-item-author-bar mt-1">
+            <div class="profile-box-trigger">
```

Comment:
Well, maybe not required in this PR, but this jQuery trigger class isn't really required here at all. We need Vue to watch for a click even on this div instead and then change some component state to make `profile-box` below visible. The same, `profile-box` class `hidden` should only render if this component local state defines that it should be hidden.